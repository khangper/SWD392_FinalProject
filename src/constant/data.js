export const API_KEY = "API_KEY";

export const FETCH_HOME_LIVESTREAM_REQUEST = "FETCH_HOME_LIVESTREAM_REQUEST";
export const FETCH_HOME_LIVESTREAM_SUCCESS = "FETCH_HOME_LIVESTREAM_SUCCESS";
export const FETCH_HOME_LIVESTREAM_FAILURE = "FETCH_HOME_LIVESTREAM_FAILURE";

export const FETCH_HOME_FEATURED_COURSES_REQUEST =
  "FETCH_HOME_FEATURED_COURSES_REQUEST";
export const FETCH_HOME_FEATURED_COURSES_SUCCESS =
  "FETCH_HOME_FEATURED_COURSES_SUCCESS";
export const FETCH_HOME_FEATURED_COURSES_FAILURE =
  "FETCH_HOME_FEATURED_COURSES_FAILURE";
// All Live Stream
export const FETCH_STREAMS_REQUEST = "FETCH_STREAMS_REQUEST";
export const FETCH_STREAMS_SUCCESS = "FETCH_STREAMS_SUCCESS";
export const FETCH_STREAMS_FAILURE = "FETCH_STREAMS_FAILURE";

//All Instructor
export const FETCH_INSTRUCTORS_REQUEST = "FETCH_INSTRUCTORS_REQUEST";
export const FETCH_INSTRUCTORS_SUCCESS = "FETCH_INSTRUCTORS_SUCCESS";
export const FETCH_INSTRUCTORS_FAILURE = "FETCH_INSTRUCTORS_FAILURE";
export const SEARCH_INSTRUCTORS_REQUEST = "SEARCH_INSTRUCTORS_REQUEST";
export const FETCH_INSTRUCTOR_BY_ID_REQUEST = "FETCH_INSTRUCTOR_BY_ID_REQUEST";

// SaveCourse.js
export const FETCH_SAVECOURSES_REQUEST = "FETCH_SAVECOURSES_REQUEST";
export const FETCH_SAVECOURSES_SUCCESS = "FETCH_SAVECOURSES_SUCCESS";
export const FETCH_SAVECOURSES_FAILURE = "FETCH_SAVECOURSES_FAILURE";
export const DELETE_SAVECOURSE_REQUEST = "DELETE_SAVECOURSE_REQUEST";
export const DELETE_SAVECOURSE_SUCCESS = "DELETE_SAVECOURSE_SUCCESS";
export const DELETE_SAVECOURSE_FAILURE = "DELETE_SAVECOURSE_FAILURE";

// Shoppingcart.js
export const ShoppingCart_API_URL =
  "https://66908916c0a7969efd9c67ed.mockapi.io/ojt-repo/ShoppingCart";
export const ADD_TO_CART = "ADD_TO_CART";
export const REMOVE_FROM_CART = "REMOVE_FROM_CART";
export const APPLY_DISCOUNT = "APPLY_DISCOUNT";
export const ADD_TO_CART_FAILURE = " ADD_TO_CART_FAILURE";
export const ADD_TO_CART_SUCCESS = "APPLY_DISCOUNT";

//MyCourse_Course
export const FETCH_COURSES_REQUEST = "FETCH_COURSES_REQUEST";
export const FETCH_COURSES_SUCCESS = "FETCH_COURSES_SUCCESS";
export const FETCH_COURSES_FAILURE = "FETCH_COURSES_FAILURE";

export const EDIT_COURSE_REQUEST = "EDIT_COURSE_REQUEST";
export const EDIT_COURSE_SUCCESS = "EDIT_COURSE_SUCCESS";
export const EDIT_COURSE_FAILURE = "EDIT_COURSE_FAILURE";

export const DELETE_COURSE_REQUEST = "DELETE_COURSE_REQUEST";
export const DELETE_COURSE_SUCCESS = "DELETE_COURSE_SUCCESS";
export const DELETE_COURSE_FAILURE = "DELETE_COURSE_FAILURE";

//UpcommingCourse_Course
export const FETCH_UPCOMMINGCOURSES_REQUEST = "FETCH_UPCOMMINGCOURSES_REQUEST";
export const FETCH_UPCOMMINGCOURSES_SUCCESS = "FETCH_UPCOMMINGCOURSES_SUCCESS";
export const FETCH_UPCOMMINGCOURSES_FAILURE = "FETCH_UPCOMMINGCOURSES_FAILURE";

export const EDIT_UPCOMMINGCOURSE_REQUEST = "EDIT_UPCOMMINGCOURSE_REQUEST";
export const EDIT_UPCOMMINGCOURSE_SUCCESS = "EDIT_UPCOMMINGCOURSE_SUCCESS";
export const EDIT_UPCOMMINGCOURSE_FAILURE = "EDIT_UPCOMMINGCOURSE_FAILURE";

export const DELETE_UPCOMMINGCOURSE_REQUEST = "DELETE_UPCOMMINGCOURSE_REQUEST";
export const DELETE_UPCOMMINGCOURSE_SUCCESS = "DELETE_UPCOMMINGCOURSE_SUCCESS";
export const DELETE_UPCOMMINGCOURSE_FAILURE = "DELETE_UPCOMMINGCOURSE_FAILURE";
