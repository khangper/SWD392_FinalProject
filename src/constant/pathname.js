export const PATH_NAME = {
  HOME: "/home",
  EXPLORE: "/explore",
  LIVE_OUTPUT: "/live_output/:id",
  ADD_LIVE_STREAM: "/add_livestream",
  ALL_LIVE_STREAM: "/all_livestream",
  ALL_INSTRUCTOR_LIST: "/allinstructorlist/:id",
  HELP: "/help",
  REPORT_HISTORY: "/reporthistory",
  SEND_FEEDBACK: "/sendfeedback",
  OTHER_INSTRUCTOR_VIEW: "/other-instructor-view/:id",
  MY_INSTRUCTOR_PROFILE: "/my-instructor-profile",
  SAVE_COURSES: "/save-courses",
  SETTING: "/setting",
  MY_COURSE: "/courses",
  LOGIN: "/login",
  FORGOT_PASSWORD: "/forgot-password",
  SIGN_UP: "/sign-up",
  SIGN_UP_STEP: "/SignUpStep",
  THANKS: "/thanks",
  ERROR_404: "/error404",
  INVOICE: "/invoice",
  PRESS: "/press",
  SEARCH_RESULT: "/search_result",
  PAID_MEMBERSHIP: "/paidmembership",
  ABOUT_US: "/about-us",
  OUR_BLOG: "/our-blog",
  BLOG_SINGLE_VIEW: "/blog-single-view",
  CHECKOUT: "/checkout",
  SHOPPING_CART: "/shoppingcart",
  HELP_VIEW_1: "/helpview1",
  HELP_VIEW_2: "/helpview2",
  COMPANY_DETAILS: "/company-details",
  CAREER: "/career",
  APPLY_JOB: "/apply-job",
  CONTACT_US: "/contactus",
  TERMS_OF_USE: "/termsofuse",
  CERTIFICATION_FILL_FORM: "/certificationfillform",
  CERTIFICATION_TEST: "/certificates-test",
  TEST_RESULT: "/test-result",
  MESSAGE: "/messages",
  COURSES_DETAIL_VIEW: "/courses-detail-view/:id",
  PREVIEWS: "/reviews",
  COMING_SOON: "/coming-soon",
  INS_DASHBOARD: "/ins-dashboard",
  PURCHASE_COURSE: "/PurchaseCourse",
  NOTIFICATON: "/notifications",
  DOWLOADCOURSE: "/dowloadcourse",
  CERTIFICATION_CENTER: "/certificationcenter",
  SITEMAP: "/sitemap",
  EARNING: "/earning",
  PAYOUT: "/payout",
  VERTIFICATION: "/vertification",
  MYCERTIFICATIONS: "/mycertifications",
  CREDIT: "/credits",
  STATEMENTIS: "/statementis",
  STATEMENTST: "/statementst",
  ANALYTICS: "/analytics",
  CREATE_NEW_COURSE: "/create-course",
  STD_DASHBOARD: "/student-dashboard",
  NOTIFICATIONSTUDENT: "/notificationsstudent",
};
