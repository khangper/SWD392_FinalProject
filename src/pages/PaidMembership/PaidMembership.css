._215b15 {
    display: grid;
    grid-template-columns: 1fr; /* Sử dụng CSS Grid thay vì Bootstrap col */
    width: 58%;
    padding: 20px 0 0;
    background: #fff;
    margin-bottom: 17px;
    justify-items: left; /* Đặt các mục vào giữa theo chiều ngang */
}

.title125 {
    width: 100%;
    float: none; /* Bỏ float để nội dung căn giữa */
    text-align: left; /* Căn giữa nội dung */
}

.titleleft {
    float: none; /* Bỏ float để nội dung căn giữa */
    text-align: left; /* Căn giữa nội dung */
}

.ttl121 .breadcrumb {
    background-color: #fff !important;
    padding: 0 !important;
    text-align: center; /* Căn giữa các breadcrumb */
}

.ttl121 .breadcrumb-item a {
    color: #333;
}

.ttl121 .breadcrumb-item {
    font-size: 14px;
    font-weight: 400;
    color: #333;
    font-family: 'Roboto', sans-serif;
    line-height: 24px;
}

.Paidmember-title {
    text-align: center; /* Căn giữa tiêu đề */
    margin-top: 34px;
}

.Paidmember-title h2 {
    font-size: 24px;
    font-weight: 500;
    font-family: 'Roboto', sans-serif;
    color: #333;
    text-align: center; /* Căn giữa tiêu đề */
    line-height: 30px;
    position: sticky;
    top: 0; /* Đặt vị trí dính (sticky) từ đầu trang */
}

.PMMS-container {
  background: #f7f7f7;
  padding-left: 19%;
  padding-right: 6%;
  padding-top: 20px;
  padding-bottom: 20px;
  transition: padding-left 0.4s;
  overflow-y: auto;
  flex: 1 1;
}


/**/

.membership-container {
    display: grid;
    grid-template-columns: 1fr 1fr;
    gap: 20px;
}

.membership-bg {
    background-color: #f9f9f9;
    padding: 20px;
    border-radius: 8px;
    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
}

.membership-title {
    display: flex;
    justify-content: space-between;
    align-items: center;
    margin-bottom: 20px;
}

.membership-left {
    flex: 1;
}

.membership-price {
    font-size: 1.5rem;
    color: #333;
}

.save-price {
    font-size: 0.9rem;
    color: #999;
}

.membership-right img {
    width: 50px;
    height: 50px;
}

.membership-des {
    margin-bottom: 20px;
}

.prmb-btn {
    display: inline-block;
    padding: 10px 20px;
    background-color: #007bff;
    color: white;
    text-align: center;
    border: none;
    border-radius: 4px;
    cursor: pointer;
}

.prmb-btn:hover {
    background-color: #0056b3;
}


.membership-container {
    display: grid;
    grid-template-columns: 1fr 1fr;
    gap: 20px;
    padding: 82px;
    margin-right: 144px;
}
  .membership-card {
    background: #fff;
    width: 100%;
    float: left;
    padding: 30px;
    border-radius: 3px;
    border: 1px solid #efefef;
    box-shadow: -0.083333333in 0.5pc 1.125pc -7px rgba(25, 42, 70, .13);
    transition: all .2s ease-in-out;
    margin-top: 47px;
  }


  .membership-title {
    display: flex;
    justify-content: space-between;
    align-items: center;
    margin-bottom: 15px;
  }
  .membership-title h2 {
    margin: 0;
  }
  .membership-price {
    font-size: 18px;
    font-weight: 500;
    font-family: 'Roboto', sans-serif;
    margin-bottom: 10px !important;
    color: #ed2a26;
    text-align: left;
    line-height: 26px;
  }


  .save-price {
    font-size: 0.9em;
    color: #777;
  }
  .membership-des {
    font-size: 16px;
    font-weight: 400;
    font-family: 'Roboto', sans-serif;
    margin-top: 40px !important;
    color: #686f7a;
    text-align: left;
    line-height: 26px;
}
  
  .accordion {
    margin-bottom: 15px;
  }
  details {
    border: 1px solid #ddd;
    border-radius: 5px;
    margin-bottom: 10px;
  }
  summary {
    padding: 10px;
    cursor: pointer;
    display: flex;
    font-size: 18px;
    font-weight: 500;
    color: #333;
    font-family: 'Roboto', sans-serif;
    /* justify-content: space-between; */
  }



  .faq-item summary {
    font-size: 18px;
    font-weight: 500;
    color: #333;
    font-family: 'Roboto', sans-serif;
    padding: 23px 16px 18px;
    display: block;
    line-height: 26px;
    cursor: pointer; /* Optional: add pointer cursor for better usability */
}

  summary i {
    margin-right: 10px;
  }
  .panel-body {
    font-size: 14px;
    font-weight: 400;
    font-family: 'Roboto', sans-serif;
    color: #686f7a;
    line-height: 26px;
    padding-bottom: 20px;
    text-align: left;
  }
  .prmb_btn {
    margin-top: 20px;
    width: 100%;
    height: 50px;
    border-radius: 3px;
    border: 0;
    background: #ed2a26;
    color: #fff;
    font-size: 14px;
    font-weight: 500;
    font-family: 'Roboto', sans-serif;
}
  .prmb_btn:hover {
    background-color: #313437;
  }
  .chck_icon_1 {
    color: red;
  }
  .Member-icon {
    width: 69px;
}

.Check-iocn {
    width: 18px;
    margin-right: 7px;
}


.Paidmember1-grid-container {
  display: grid;
  grid-template-columns: 1fr;
  gap: 20px;
  margin-top: 62px;
  margin-left: -160px;
}

.title-section {
  text-align: center;
}

.faq-section {
  margin-top: -215px;
  padding: 241px;
  display: grid;
  gap: 10px;
}

.panel {
  border: 1px solid #ddd;
  border-radius: 4px;
  overflow: hidden;
}

.panel-heading {
  background-color: #f7f7f7;
  padding: 10px 15px;
}

.panel-title a {
  text-decoration: none;
  display: block;
  color: #333;
}

.panel-body {
  padding: 15px;
  background-color: #fff;
}