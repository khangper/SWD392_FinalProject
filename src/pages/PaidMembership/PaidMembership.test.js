import React from "react";
import { render, screen, fireEvent } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import { renderWithProviders } from "../../constant/test-util";
import PaidMembership from "./PaidMembership";

describe("PaidMembership", () => {
  beforeAll(() => {
    jest.spyOn(console, "error").mockImplementation(() => {});
    jest.spyOn(console, "log").mockImplementation(() => {});
  });

  afterAll(() => {
    console.error.mockRestore();
    console.log.mockRestore();
  });

  it("renders the PaidMembership component", () => {
    const { container } = renderWithProviders(<PaidMembership />);
    expect(container).toBeDefined();
    expect(container).toMatchSnapshot();
  });

  it("displays the membership plans correctly", () => {
    renderWithProviders(<PaidMembership />);

    // Check if the membership plans are rendered
    expect(screen.getByText(/Baby Plan/i)).toBeInTheDocument();
    expect(screen.getByText(/\$49\/month/i)).toBeInTheDocument();
    expect(
      screen.getByText(/Save \$79 when paid annually/i)
    ).toBeInTheDocument();

    expect(screen.getByText(/Business Plan/i)).toBeInTheDocument();
    expect(screen.getByText(/\$99\/month/i)).toBeInTheDocument();
    expect(
      screen.getByText(/Save \$189 when paid annually/i)
    ).toBeInTheDocument();
  });

  it("renders all accordion items closed by default", () => {
    renderWithProviders(<PaidMembership />);

    // Select all details elements
    const detailsElements = screen.getAllByRole("group");

    // Ensure each details element is not open
    detailsElements.forEach((details) => {
      expect(details).not.toHaveAttribute("open");
    });
  });

  it("toggles accordion items when clicked", () => {
    renderWithProviders(<PaidMembership />);

    // Select the first details element
    const firstDetails = screen.getAllByRole("group")[0];
    const firstSummary = firstDetails.querySelector("summary");

    // Initially closed
    expect(firstDetails).not.toHaveAttribute("open");

    // Click to open
    fireEvent.click(firstSummary);
    expect(firstDetails).toHaveAttribute("open");

    // Click to close
    fireEvent.click(firstSummary);
    expect(firstDetails).not.toHaveAttribute("open");
  });

  it("does not render the 'Exclusive Offer' banner when no offers available", () => {
    renderWithProviders(<PaidMembership />);

    // Assuming there's a conditional rendering for an exclusive offer banner
    expect(screen.queryByText(/Exclusive Offer/i)).not.toBeInTheDocument();
  });
});
