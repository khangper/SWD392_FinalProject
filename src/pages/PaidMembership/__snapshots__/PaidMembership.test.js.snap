// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PaidMembership renders the PaidMembership component 1`] = `
<div>
  <div>
    <header
      class="header1"
    >
      <a
        href="/home"
      >
        <button
          class="header-link1"
        >
          Back To Cursus
        </button>
      </a>
      <a
        href="/home"
      >
        <img
          alt=""
          class="header-group-icon"
          src="https://gambolthemes.net/html-items/cursus-new-demo/images/logo.svg"
        />
      </a>
      <img
        alt=""
        class="header-avatar"
        src="profile-img.jpg"
      />
    </header>
    <main>
      <div
        class="HelpView1-grid-container"
      >
        <div
          class="HelpView-title125"
        >
          <div
            class="HelpView-titleleft"
          >
            <div
              class="HelpView-ttl121"
            >
              <nav
                aria-label="breadcrumb"
              >
                <ol
                  class="HelpView-breadcrumb"
                >
                  <li
                    class="HelpView-breadcrumb-item"
                  >
                    <a
                      href="index.html"
                    >
                      Home
                    </a>
                  </li>
                  <li
                    aria-current="page"
                    class="HelpView-breadcrumb-item active"
                  >
                    /Paymember Ship
                  </li>
                </ol>
              </nav>
            </div>
          </div>
        </div>
        <div
          class="HelpView-title126"
        >
          <h2>
            Paymember Ship
          </h2>
        </div>
      </div>
      <div
        class="memb4d25"
      >
        <div
          class="PMMS-container"
        >
          <div
            class="membership-container"
          >
            <div
              class="membership-card"
            >
              <div
                class="membership-title"
              >
                <div>
                  <h2>
                    Baby Plan
                  </h2>
                  <div
                    class="membership-price"
                  >
                    $49/month
                  </div>
                  <div
                    class="save-price"
                  >
                    Save $79 when paid annually
                  </div>
                </div>
                <img
                  alt=""
                  class="Member-icon"
                  src="https://gambolthemes.net/html-items/cursus-new-demo/images/membership/icon-1.svg"
                />
              </div>
              <div
                class="membership-des"
              >
                <p>
                  For instructors launching their first digital products.
                </p>
              </div>
              <div
                class="accordion"
              >
                <details>
                  <summary
                    class="details-summary "
                  >
                    <div
                      class="left-content"
                    >
                      <img
                        alt=""
                        class="Check-icon"
                        src="check.png"
                      />
                      <span>
                        Your own shop
                      </span>
                    </div>
                    <span
                      class="icon"
                    >
                      ✚
                    </span>
                  </summary>
                  <div
                    class="panel-body"
                  >
                    Anim pariatur cliche reprehenderit, enim eiusmod high life accusamus terry richardson ad squid. VHS.
                  </div>
                </details>
                <details>
                  <summary
                    class="details-summary "
                  >
                    <div
                      class="left-content"
                    >
                      <img
                        alt=""
                        class="Check-icon"
                        src="check.png"
                      />
                      <span>
                        Online courses
                      </span>
                    </div>
                    <span
                      class="icon"
                    >
                      ✚
                    </span>
                  </summary>
                  <div
                    class="panel-body"
                  >
                    Anim pariatur cliche reprehenderit, enim eiusmod high life accusamus terry richardson ad squid.
                  </div>
                </details>
                <details>
                  <summary
                    class="details-summary "
                  >
                    <div
                      class="left-content"
                    >
                      <img
                        alt=""
                        class="Check-icon"
                        src="check.png"
                      />
                      <span>
                        Email marketing
                      </span>
                    </div>
                    <span
                      class="icon"
                    >
                      ✚
                    </span>
                  </summary>
                  <div
                    class="panel-body"
                  >
                    Anim pariatur cliche reprehenderit, enim eiusmod high life accusamus terry richardson ad squid.
                  </div>
                </details>
                <details>
                  <summary
                    class="details-summary "
                  >
                    <div
                      class="left-content"
                    >
                      <img
                        alt=""
                        class="Check-icon"
                        src="check.png"
                      />
                      <span>
                        Messaging
                      </span>
                    </div>
                    <span
                      class="icon"
                    >
                      ✚
                    </span>
                  </summary>
                  <div
                    class="panel-body"
                  >
                    Anim pariatur cliche reprehenderit, enim eiusmod high life accusamus terry richardson ad squid. le VHS.
                  </div>
                </details>
                <details>
                  <summary
                    class="details-summary "
                  >
                    <div
                      class="left-content"
                    >
                      <img
                        alt=""
                        class="Check-icon"
                        src="check.png"
                      />
                      <span>
                        Zero charges commissions 10 sales
                      </span>
                    </div>
                    <span
                      class="icon"
                    >
                      ✚
                    </span>
                  </summary>
                  <div
                    class="panel-body"
                  >
                    Anim pariatur cliche reprehenderit, enim eiusmod high life accusamus terry richardson ad squid.
                  </div>
                </details>
                <details>
                  <summary
                    class="details-summary "
                  >
                    <div
                      class="left-content"
                    >
                      <img
                        alt=""
                        class="Check-icon"
                        src="check.png"
                      />
                      <span>
                        7-days-a-week support
                      </span>
                    </div>
                    <span
                      class="icon"
                    >
                      ✚
                    </span>
                  </summary>
                  <div
                    class="panel-body"
                  >
                    Anim pariatur cliche reprehenderit, enim eiusmod high life accusamus terry richardson ad squid.
                  </div>
                </details>
                <details>
                  <summary
                    class="details-summary1 "
                  >
                    <div
                      class="left-content"
                    >
                      <img
                        alt=""
                        class="Check-icon"
                        src="Close-paidment.png"
                      />
                      <span>
                        Memberships
                      </span>
                    </div>
                  </summary>
                </details>
                <details>
                  <summary
                    class="details-summary1 "
                  >
                    <div
                      class="left-content"
                    >
                      <img
                        alt=""
                        class="Check-icon"
                        src="Close-paidment.png"
                      />
                      <span>
                        Blog
                      </span>
                    </div>
                  </summary>
                </details>
                <details>
                  <summary
                    class="details-summary1 "
                  >
                    <div
                      class="left-content"
                    >
                      <img
                        alt=""
                        class="Check-icon"
                        src="Close-paidment.png"
                      />
                      <span>
                        Affiliate marketing
                      </span>
                    </div>
                  </summary>
                </details>
                <details>
                  <summary
                    class="details-summary1 "
                  >
                    <div
                      class="left-content"
                    >
                      <img
                        alt=""
                        class="Check-icon"
                        src="Close-paidment.png"
                      />
                      <span>
                        Third-party code
                      </span>
                    </div>
                  </summary>
                </details>
              </div>
              <button
                class="prmb_btn1"
              >
                Purchase Membership
              </button>
            </div>
            <div
              class="membership-card"
            >
              <div
                class="membership-title"
              >
                <div>
                  <h2>
                    Business Plan
                  </h2>
                  <div
                    class="membership-price"
                  >
                    $99/month
                  </div>
                  <div
                    class="save-price"
                  >
                    Save $189 when paid annually
                  </div>
                </div>
                <img
                  alt=""
                  class="Member-icon"
                  src="https://gambolthemes.net/html-items/cursus-new-demo/images/membership/icon-2.svg"
                />
              </div>
              <div
                class="membership-des"
              >
                <p>
                  For instructors who are ready to grow their business.
                </p>
              </div>
              <div
                class="accordion"
              >
                <details>
                  <summary
                    class="details-summary "
                  >
                    <div
                      class="left-content"
                    >
                      <img
                        alt=""
                        class="Check-icon"
                        src="check.png"
                      />
                      <span>
                        Your own shop
                      </span>
                    </div>
                    <span
                      class="icon"
                    >
                      ✚
                    </span>
                  </summary>
                  <div
                    class="panel-body"
                  >
                    Anim pariatur cliche reprehenderit, enim eiusmod high life accusamus terry richardson ad squid. VHS.
                  </div>
                </details>
                <details>
                  <summary
                    class="details-summary "
                  >
                    <div
                      class="left-content"
                    >
                      <img
                        alt=""
                        class="Check-icon"
                        src="check.png"
                      />
                      <span>
                        Online courses
                      </span>
                    </div>
                    <span
                      class="icon"
                    >
                      ✚
                    </span>
                  </summary>
                  <div
                    class="panel-body"
                  >
                    Anim pariatur cliche reprehenderit, enim eiusmod high life accusamus terry richardson ad squid.
                  </div>
                </details>
                <details>
                  <summary
                    class="details-summary "
                  >
                    <div
                      class="left-content"
                    >
                      <img
                        alt=""
                        class="Check-icon"
                        src="check.png"
                      />
                      <span>
                        Email marketing
                      </span>
                    </div>
                    <span
                      class="icon"
                    >
                      ✚
                    </span>
                  </summary>
                  <div
                    class="panel-body"
                  >
                    Anim pariatur cliche reprehenderit, enim eiusmod high life accusamus terry richardson ad squid.
                  </div>
                </details>
                <details>
                  <summary
                    class="details-summary "
                  >
                    <div
                      class="left-content"
                    >
                      <img
                        alt=""
                        class="Check-icon"
                        src="check.png"
                      />
                      <span>
                        Messaging
                      </span>
                    </div>
                    <span
                      class="icon"
                    >
                      ✚
                    </span>
                  </summary>
                  <div
                    class="panel-body"
                  >
                    Anim pariatur cliche reprehenderit, enim eiusmod high life accusamus terry richardson ad squid. le VHS.
                  </div>
                </details>
                <details>
                  <summary
                    class="details-summary "
                  >
                    <div
                      class="left-content"
                    >
                      <img
                        alt=""
                        class="Check-icon"
                        src="check.png"
                      />
                      <span>
                        Zero charges commissions 10 sales
                      </span>
                    </div>
                    <span
                      class="icon"
                    >
                      ✚
                    </span>
                  </summary>
                  <div
                    class="panel-body"
                  >
                    Anim pariatur cliche reprehenderit, enim eiusmod high life accusamus terry richardson ad squid.
                  </div>
                </details>
                <details>
                  <summary
                    class="details-summary "
                  >
                    <div
                      class="left-content"
                    >
                      <img
                        alt=""
                        class="Check-icon"
                        src="check.png"
                      />
                      <span>
                        7-days-a-week support
                      </span>
                    </div>
                    <span
                      class="icon"
                    >
                      ✚
                    </span>
                  </summary>
                  <div
                    class="panel-body"
                  >
                    Anim pariatur cliche reprehenderit, enim eiusmod high life accusamus terry richardson ad squid.
                  </div>
                </details>
                <details>
                  <summary
                    class="details-summary "
                  >
                    <div
                      class="left-content"
                    >
                      <img
                        alt=""
                        class="Check-icon"
                        src="check.png"
                      />
                      <span>
                        Memberships
                      </span>
                    </div>
                    <span
                      class="icon"
                    >
                      ✚
                    </span>
                  </summary>
                  <div
                    class="panel-body"
                  >
                    Anim pariatur cliche reprehenderit, enim eiusmod high life accusamus terry richardson ad squid.
                  </div>
                </details>
                <details>
                  <summary
                    class="details-summary "
                  >
                    <div
                      class="left-content"
                    >
                      <img
                        alt=""
                        class="Check-icon"
                        src="check.png"
                      />
                      <span>
                        Memberships
                      </span>
                    </div>
                    <span
                      class="icon"
                    >
                      ✚
                    </span>
                  </summary>
                  <div
                    class="panel-body"
                  >
                    Anim pariatur cliche reprehenderit, enim eiusmod high life accusamus terry richardson ad squid.
                  </div>
                </details>
                <details>
                  <summary
                    class="details-summary "
                  >
                    <div
                      class="left-content"
                    >
                      <img
                        alt=""
                        class="Check-icon"
                        src="check.png"
                      />
                      <span>
                        Affiliate marketing
                      </span>
                    </div>
                    <span
                      class="icon"
                    >
                      ✚
                    </span>
                  </summary>
                  <div
                    class="panel-body"
                  >
                    Anim pariatur cliche reprehenderit, enim eiusmod high life accusamus terry richardson ad squid.
                  </div>
                </details>
                <details>
                  <summary
                    class="details-summary "
                  >
                    <div
                      class="left-content"
                    >
                      <img
                        alt=""
                        class="Check-icon"
                        src="check.png"
                      />
                      <span>
                        Third-party code
                      </span>
                    </div>
                    <span
                      class="icon"
                    >
                      ✚
                    </span>
                  </summary>
                  <div
                    class="panel-body"
                  >
                    Anim pariatur cliche reprehenderit, enim eiusmod high life accusamus terry richardson ad squid.
                  </div>
                </details>
              </div>
              <button
                class="prmb_btn"
              >
                Purchase Membership
              </button>
            </div>
          </div>
          <div
            class="Paidmember1-grid-container"
          >
            <div
              class="title-section"
            >
              <h2>
                Membership FAQ
              </h2>
              <p>
                Wait, what about…
              </p>
            </div>
            <div
              class="faq-section"
            >
              <details
                class="faq-item"
              >
                <summary>
                  Is it easy to change plans?
                </summary>
                <div
                  class="panel-body"
                >
                  Anim pariatur cliche reprehenderit, enim eiusmod high life accusamus terry richardson ad squid.le VHS.
                </div>
              </details>
              <details
                class="faq-item"
              >
                <summary>
                  Can I cancel at any time?
                </summary>
                <div
                  class="panel-body"
                >
                  Anim pariatur cliche reprehenderit, enim eiusmod high life accusamus terry richardson ad squid.
                </div>
              </details>
              <details
                class="faq-item"
              >
                <summary>
                  Are there any credit card or PayPal fees?
                </summary>
                <div
                  class="panel-body"
                >
                  Anim pariatur cliche reprehenderit, enim eiusmod high life accusamus terry richardson ad squid.
                </div>
              </details>
              <details
                class="faq-item"
              >
                <summary>
                  Do I have to choose my plan before I start my trial?
                </summary>
                <div
                  class="panel-body"
                >
                  Anim pariatur cliche reprehenderit, enim eiusmod high life accusamus terry richardson ad squid.
                </div>
              </details>
              <details
                class="faq-item"
              >
                <summary>
                  Edututs+ isn’t for me. Can I have a refund?
                </summary>
                <div
                  class="panel-body"
                >
                  Anim pariatur cliche reprehenderit, enim eiusmod high life accusamus terry richardson ad squid.
                </div>
              </details>
              <details
                class="faq-item"
              >
                <summary>
                  How do I get my money? 
                </summary>
                <div
                  class="panel-body"
                >
                  Anim pariatur cliche reprehenderit, enim eiusmod high life accusamus terry richardson ad squid.
                </div>
              </details>
              <details
                class="faq-item"
              >
                <summary>
                  Do I need a credit card to sign up?
                </summary>
                <div
                  class="panel-body"
                >
                  Anim pariatur cliche reprehenderit, enim eiusmod high life accusamus terry richardson ad squid.
                </div>
              </details>
              <details
                class="faq-item"
              >
                <summary>
                  Is Edututs+ safe and secure for online transactions?
                </summary>
                <div
                  class="panel-body"
                >
                  Anim pariatur cliche reprehenderit, enim eiusmod high life accusamus terry richardson ad squid.
                </div>
              </details>
              <details
                class="faq-item"
              >
                <summary>
                  I still have questions. HELP!
                </summary>
                <div
                  class="panel-body"
                >
                  Anim pariatur cliche reprehenderit, enim eiusmod high life accusamus terry richardson ad squid.
                </div>
              </details>
            </div>
          </div>
        </div>
      </div>
    </main>
    <footer
      class="footer footer-large-container"
    >
      <div
        class="footer-links"
      >
        <div
          class="footer-column"
        >
          <a
            class="link"
            href="/about-us"
          >
            About
          </a>
          <a
            class="link"
            href="/blog-single-view/:id"
          >
            Blog
          </a>
          <a
            class="link"
            href="/career"
          >
            Careers
          </a>
          <a
            class="link"
            href="/press"
          >
            Press
          </a>
        </div>
        <div
          class="footer-column"
        >
          <a
            class="link"
            href="/help"
          >
            Help
          </a>
          <a
            class="link"
            href="/coming-soon"
          >
            Advertise
          </a>
          <a
            class="link"
            href="/coming-soon"
          >
            Developers
          </a>
          <a
            class="link"
            href="/contactus"
          >
            Contact Us
          </a>
        </div>
        <div
          class="footer-column"
        >
          <a
            class="link"
            href="/termsofuse"
          >
            Copyright Policy
          </a>
          <a
            class="link"
            href="/termsofuse"
          >
            Terms
          </a>
          <a
            class="link"
            href="/termsofuse"
          >
            Privacy Policy
          </a>
          <a
            class="link"
            href="/sitemap"
          >
            Sitemap
          </a>
        </div>
        <div
          class="footer-column"
        >
          <button
            class="teach-button"
          >
            Teach on Cursus
          </button>
          <div
            class="language-selector"
          >
            <button>
              Language 
              <span>
                ▼
              </span>
            </button>
          </div>
        </div>
      </div>
      <div
        class="footer-bottom"
      >
        <div
          class="footer-copy"
        >
          <img
            alt=""
            src="https://gambolthemes.net/html-items/cursus-new-demo/images/logo1.svg"
          />
          © 2024 
          <strong>
            Cursus
          </strong>
          . All Rights Reserved.
        </div>
        <div
          class="footer-social"
        >
          <a
            href="/home"
          >
            <img
              alt=""
              class="social-icon"
              src="facebook.png"
            />
          </a>
          <a
            href="/home"
          >
            <img
              alt=""
              class="social-icon"
              src="x.png"
            />
          </a>
          <a
            href="/home"
          >
            <img
              alt=""
              class="social-icon"
              src="google+.png"
            />
          </a>
          <a
            href="/home"
          >
            <img
              alt=""
              class="social-icon"
              src="linkedin.png"
            />
          </a>
          <a
            href="/home"
          >
            <img
              alt=""
              class="social-icon"
              src="insta.png"
            />
          </a>
          <a
            href="/home"
          >
            <img
              alt=""
              class="social-icon"
              src="pinterest.png"
            />
          </a>
        </div>
      </div>
    </footer>
  </div>
</div>
`;
